partner_delivery_multicompany_default/tests/test_partner.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2023 Moduon Team S.L.
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class PartnerDefaultCarrierCase(AccountTestInvoicingCommon):
```

Comment:
This base class is not turning off tracking. Could you please do it for all the test classes?

Reply:
Done